<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="3">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Thermometer 4</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditThermometer4"/>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBoxThermometer4">
         <property name="maximum">
          <number>125</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="horizontalSliderThermometer4">
         <property name="maximum">
          <number>125</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonThermometer4">
         <property name="text">
          <string>Enable/Disable</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Thermometer 1</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditThermometer1"/>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBoxThermometer1">
         <property name="maximum">
          <number>125</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="horizontalSliderThermometer1">
         <property name="maximum">
          <number>125</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonThermometer1">
         <property name="text">
          <string>Enable/Disable</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Thermometer 2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditThermometer2"/>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="horizontalSliderThermometer2">
         <property name="maximum">
          <number>125</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBoxThermometer2">
         <property name="maximum">
          <number>125</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonThermometer2">
         <property name="text">
          <string>Enable/Disable</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Thermometer 3</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditThermometer3"/>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="horizontalSliderThermometer3">
         <property name="maximum">
          <number>125</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBoxThermometer3">
         <property name="maximum">
          <number>125</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonThermometer3">
         <property name="text">
          <string>Enable/Disable</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connections</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxDevices"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonDisconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Logs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditLogs"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
